app/presenters/business_presenter.rb

```diff
@@ -0,0 +1,11 @@
+module BusinessPresenter
+
+  def display_phone
+    self.raw_data['display_phone'] || self.raw_data['display_phone'] = "No Number Provided"
+  end
+
+  def image_url
+    self.raw_data['image_url'] || self.raw_data['image_url'] = "socialteering_placeholder.png"
```

Comment:
Line is too long. [94/80]<br>Redundant `self` detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.